src/tools/cephfs/top/cephfs-top

```diff
@@ -811,7 +813,9 @@ class FSTop(object):
                                                                  num_mounts=num_mounts,
                                                                  num_kclients=num_kclients,
                                                                  num_libs=num_libs), curses.A_DIM)
-        self.header.addstr(4, 0, help, curses.A_DIM)
+        self.header.addstr(4, 0, f"Filters: Sort={current_states['last_field']}, "
```

Comment:
It seems helpful from user's perspective, but making the header look a bit clumsy.(not having any strong opinion, may be some other way or place :|).
![header](https://user-images.githubusercontent.com/79842675/202422383-e495ec8e-9ca0-44eb-a231-a8cc143b259f.png)

Reply:
Another way is to have different colour indications for sort & limit. But then it couldn't show the sorted field.
About another place, I think this is the only place in `Info` screens to display the current status. May be, we could make it the same line as `Total Clients` ?

Reply:
> Another way is to have different colour indications for sort & limit. But then it couldn't indicate the sorted field. About another place, I think this is the only place in `Info` screens to display the current status. May be, we could make it the same line as `Total Clients` ?

I think there is no better way. Let's keep it as it is, just with small change : s/ Sort=chit, Limit=None/Sort - chit , Limit - None

Reply:
Considering the limitations of what ncurses provides, this way of displaying the settings seems better and much more informative than the alternatives we've considered. Updated the PR as per your comment.